Hw_7_Sotnikov.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+from math import ceil
+
+
+# # Pandas intersect
+
+
+# Functions for reading .GFF and .BED files
+
+def read_gff(path_to_file):
+    gff = pd.read_csv(path_to_file, sep="\t", names=["chrom", "source", "type", "start", "end", "score", "strand", "phase", "attributes"], header=0)
+    return gff
+
+def read_bed(path_to_file):
+    bed = pd.read_csv(path_to_file, sep="\t", names=["chrom", "chromStart", "chromEnd", "name", "bed_score", "bed_strand"])
+    return bed
```

Comment:
Твой код учитывает только случай, где комментарий занимает только первую строку. Однако чаще всего в файлах данного формата могут быть комментарии на ммного строк, их нужно учитывать. Благо в пандасе это делается супер просто
```suggestion
def read_gff(path_to_file):
    gff = pd.read_csv(path_to_file, sep="\t", names=["chrom", "source", "type", "start", "end", "score", "strand", "phase", "attributes"], header=0, comment="#")
    return gff

def read_bed(path_to_file):
    bed = pd.read_csv(path_to_file, sep="\t", names=["chrom", "chromStart", "chromEnd", "name", "bed_score", "bed_strand"], comment="#")
    return bed
```